### PR TITLE
Toolbar improvements (connect #668)

### PR DESCRIPTION
**Pull request by valllllll2000**
#### Before the PR (what is the issue or what needed to be done)
using old action bar + action bar very ugly
#### The solution
moved to new toolbar, updated styling as specified in the issue
#### Screenshots (if appropriate)
see issue
#### Reviewer Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: app/src/main/java/org/akvo/flow/activity/SurveyActivity.java [hunk 1]
```diff
@@ -83,6 +84,7 @@
     private DrawerLayout mDrawerLayout;
     private ActionBarDrawerToggle mDrawerToggle;
     private DrawerFragment mDrawer;
+    private Toolbar toolbar;
```
Comment (nhternup) [inline app/src/main/java/org/akvo/flow/activity/SurveyActivity.java]: Should this class also extend BackActivity and call setupToolBar()

File: app/src/main/res/layout/tab_layout.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<android.support.design.widget.TabLayout
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:id="@+id/tabs"
+        android:layout_gravity="top"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/orange_main"
+        app:tabBackground="@color/orange_main"
```
Comment (nhternup) [inline app/src/main/res/layout/tab_layout.xml]: tabBackground may not be required

File: app/src/main/res/layout/form_activity.xml [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+        <android.support.design.widget.TabLayout
+                android:id="@+id/tabs"
+                android:layout_gravity="top"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@color/orange_main"
+                app:tabBackground="@color/orange_main"
```
Comment (nhternup) [inline app/src/main/res/layout/form_activity.xml]: tabBackground may not be required here

File: app/src/main/java/org/akvo/flow/activity/BackActivity.java [hunk 4]
```diff
@@ -19,21 +19,21 @@
...
+import android.support.v7.widget.Toolbar;
 import android.view.MenuItem;
 
-public abstract class BackActivity extends AppCompatActivity {
+import org.akvo.flow.R;
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
+public abstract class BackActivity extends AppCompatActivity {
```
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/activity/BackActivity.java]: This class contains the back functionality, but the base activity is for injection, they will not be merged, maybe this one will extend BaseActivity but I am not sure yet.

File: app/src/main/java/org/akvo/flow/activity/SurveyActivity.java (hunk 1 above)
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/activity/SurveyActivity.java]: No, this one contains navigation drawer and not the back functionality

**Comment by valllllll2000** [comment]
@nhternup I have replied and made some changes, thanks

**Pull request merged**